calus.js

```diff
@@ -22,7 +23,7 @@ let defaultTemplate = `
                 ‹
             </button>
             <div class="month__text">
-                {{ month.time.toFormat(month.isInCurrentYear ? 'MMMM' : 'MMMM y') }}
+                {{ translate('month_' + month.time.toFormat('MMMM'), 'cy') }} {{ month.time.toFormat(month.isInCurrentYear ? '' : 'y') }}
```

Comment:
You're passing `cy` in but what about when it's something different

Reply:
Actually, I will force it to default to `English` if there are no language. Language will always be passed from the SS template.